doc/index.rst

```diff
@@ -40,6 +40,7 @@ Contents
     editor
     ipythonconsole
     variableexplorer
+    plots
     help
```

Comment:
```suggestion
    help
    plots
```

It makes more sense to put the plots pane after the 4 core panes, not splitting them up.